toolchain/sem_ir/ids.h

```diff
@@ -155,18 +159,25 @@ struct ConstantId : public IdBase, public Printable<ConstantId> {
   // logic here. LLVM should still optimize this.
   static constexpr auto Abs(int32_t i) -> int32_t { return i > 0 ? i : -i; }
 
-  // Returns the instruction that describes this constant value, or
+  // Returns the instruction that describes this template constant value, or
   // InstId::Invalid for a runtime value. This is not part of the public
```

Comment:
Should this say "non-template constant" instead of "runtime value"?

Reply:
This `CHECK`-fails if it's not a template constant value; comment updated to say so.